.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*.*.*'
```

Comment:
will this appropriately handle "vX.Y.Z" as well as "X.Y.Z" are we definitely enforcing one version over the other?

Reply:
it handles both. I don't see a need to enforce that hear, since it only needs to be human readable